packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/selectionTypes/takeEveryLine.yml

```diff
@@ -18,8 +20,8 @@ initialState:
     d   e
     f   g
   selections:
-    - anchor: {line: 6, character: 0}
-      active: {line: 6, character: 0}
+    - anchor: {line: 5, character: 0}
+      active: {line: 5, character: 0}
```

Comment:
Any idea what's going on here?

Reply:
No when I test I get this new behavior for both the old implementation of take every line and the new one for take every line file. Looks like the test was just wrong somehow?

Reply:
Oh wait a minute. This is the initial state 😅